modules/cats/src/main/tut/README.md

```diff
@@ -0,0 +1,41 @@
+# Cats module for PureConfig
+
+Adds support for selected [cats](http://typelevel.org/cats/) classes to PureConfig.
+
+## Add pureconfig-cats to your project
+
+In addition to [core pureconfig](https://github.com/melrief/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-cats" % "0.7.1"
+```
+
+## Example
+
+To load a `NonEmptyList[Int]` into a configuration, we need a class to hold our configuration:
```

Comment:
I think the examples are well done and similar to the other modules 😄 . In the examples we want to show one field of the new type provided inside a configuration because every real-world usage of our library will end up having a "root" case class object. I understand this can be a bit unclear but I don't think there is a good example covering everything and if I have to choose I prefer to choose the example with a root config class and one field.